migrations/000001_init.up.sql

```diff
@@ -0,0 +1,69 @@
+CREATE TABLE clients (
```

Comment:
What do you think about [Prisma](https://www.prisma.io/docs)? I find it a great option and we wouldn't need to write raw SQL queries/schemas. The [Go client](https://goprisma.org/) makes the work pretty easy

CC @stravid87 @huzaifamk

Reply:
We already have [GORM](https://gorm.io/index.html) for query builder / ORM.

Reply:
Regarding the schema migration, I'm more familiar with golang-migrate and writing raw queries rather than using an ORM to migrate the database structure from the struct definition. I'm not sure how the library generates the database schema so i feel that it more convinient to write raw DDL.

However, I'm not against using an ORM if everyone feels it's more beneficial for managing the database schema.